client.go

```diff
@@ -0,0 +1,32 @@
+package dp_redis
```

Comment:
Double check with Eleanor on naming conventions for packages. Im not sure this follows Go idioms.

Reply:
Again you're right, it doesn't. Package names shouldn't have underscores in them. Also don't think I should use `redis` as a package name considering I'm importing a redis package too...